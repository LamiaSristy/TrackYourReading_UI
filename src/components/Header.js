import React from 'react';
import '../stylesheets/Header.css';

const Header = () => (
  <header>
    <div className="title">
      <p>Reading-Tracker</p>
    </div>
  </header>
);

export default (Header);
