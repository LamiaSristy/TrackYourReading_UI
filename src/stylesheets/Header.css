@import "https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap";

header {
  flex-shrink: 0;
  display: flex;
  background: #164663;
  font-family:
    'Poppins',
    sans-serif;
  align-items: center;
  justify-content: center;
}

.title {
  text-align: center;
  letter-spacing: 0.6px;
  font-size: 5vw;
}

.title p {
  margin: 10px;
  color: white;
}
