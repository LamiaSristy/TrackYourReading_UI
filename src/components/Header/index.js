import React from 'react';
import './Header.css';

const Header = () => (
  <header>
    <div className="title">
      <h4>TrackYourReading</h4>
    </div>
  </header>
);

export default (Header);
