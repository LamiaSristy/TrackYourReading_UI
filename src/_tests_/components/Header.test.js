import React from 'react';
import { render } from '@testing-library/react';
import Header from '../../components/Header';

describe('Header Component', () => {
  test('renders the title text of the Header Component', () => {
    const { getByText } = render(<Header />);
    const title = getByText('Reading-Tracker');
    expect(title).toBeInTheDocument();
  });

  test('renders the title text of the Header Component', () => {
    const { getByText } = render(<Header />);
    const title = getByText('Reading-Tracker');
    expect(title).toMatchSnapshot();
  });
});
