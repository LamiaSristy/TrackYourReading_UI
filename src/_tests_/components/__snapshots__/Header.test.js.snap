// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders the title text of the Header Component 1`] = `
<p>
  Reading-Tracker
</p>
`;
